poller.go

```diff
@@ -111,7 +114,7 @@ func (p *poller) manipulate(c *chunk) outcome {
 		fallthrough
 	default:
 		translateStats(output.Statistics, &c.Stats)
-		c.err = &TerminalQueryStatusError{c.queryID, status, c.stream.Text}
+		c.err = &TerminalQueryStatusError{c.queryID, string(status), c.stream.Text}
```

Comment:
Should `TerminalQueryStatusError.Status` just have the same type as `GetQueryResultsOutput.Status`?